components/web-search.tsx

```
@@ -0,0 +1,174 @@
+      <SheetTrigger asChild>
+        <Button size='sm' variant='secondary' className='rounded-full'>Sources
+          <div className='flex items-center -space-x-1.5'>
+            {results.filter(result => !!result.favicon).slice(0, 3).map(result => <img src={result.favicon} className='size-5 rounded-full border border-border bg-secondary' />)}
+          </div>
+        </Button>
+      </SheetTrigger>
+      <SheetContent className="overflow-y-auto">
+        <SheetHeader>
+          <SheetTitle>Sources</SheetTitle>
+          <SheetDescription>
+            Query: "{query}"
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)